docs/source/installing-running-locally.rst

```diff
@@ -87,11 +110,11 @@ Running Atlas Planned Maintenance Tests
 
 The ``spec-tests`` command-group is used for Atlas Planned Maintenance (APM) tests. To run a single APM test, do::
 
-  $ astrolabe spec-tests run-one <path/to/test-file.yaml> -e <path/to/workload-executor> --project-name <atlasProjectName> --cluster-name-salt <randomString>
+  $ astrolabe spec-tests run-one <path/to/test-file.yml> -e <path/to/workload-executor> --project-name <atlasProjectName> --cluster-name-salt <randomString>
```

Comment:
Looking at the existing `workload-executor` implementations, it looks like they take the connection string and path to a workload file (this filename?) as their first and second arguments, respectively. Is the workload file expected to be YAML or JSON? It looks like the Python implementation uses `yaml.load`, but Node uses `JSON.parse`.

Is it reasonable to allow each language to express its preference?

I would have expected this repo to have a Makefile that converts the YAML test files to JSON, like we do in https://github.com/mongodb/specifications/blob/master/source/Makefile.

Reply:
The workload specification is mandated to be in JSON, in https://mongodb-labs.github.io/drivers-atlas-testing/spec-workload-executor.html. I agree that the test scenario handling is somewhat awkward with the scenario being defined in YAML and then a portion of it being given to the workload executor in JSON. A jira ticket could be appropriate to look at this in the future?

Reply:
https://jira.mongodb.org/browse/DRIVERS-1563